sentinel/pkg/checker/health/app.go

```diff
@@ -45,25 +45,32 @@ func init() {
 	}
 
 	if chain == "baobab" {
-		err := json.Unmarshal(baobabJSON, &HealthCheckUrls)
+		err = json.Unmarshal(baobabJSON, &HealthCheckUrls)
 		if err != nil {
 			log.Error().Err(err).Msg("Failed to unmarshal baobab_healthcheck.json")
-			os.Exit(1)
+			return err
 		}
 	} else if chain == "cypress" {
-		err := json.Unmarshal(cypressJSON, &HealthCheckUrls)
+		err = json.Unmarshal(cypressJSON, &HealthCheckUrls)
 		if err != nil {
 			log.Error().Err(err).Msg("Failed to unmarshal cypress_healthcheck.json")
-			os.Exit(1)
+			return err
 		}
 	} else {
 		log.Error().Msg("Invalid chain")
-		os.Exit(1)
+		return err
 	}
 	log.Info().Msg("Loaded healthcheck.json")
+	return nil
 }
 
-func Start() {
+func Start() error {
+	err := setUp()
+	if err != nil {
+		log.Error().Err(err).Msg("Failed to set up health checker")
```

Comment:
this might be a duplicate error log. I think there is another similar error message on the main level

Reply:
nice catch, will be removed